src/style.css

```diff
@@ -1,3 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  --header-height: 60px;
+  --blue-color-hover: #00acdc;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+nav {
+  padding: 0;
+}
+
+.nav__list.nav__list:not(:last-child) {
+  margin-right: 20px;
+}
+
+@media (min-width: 1024px) and (max-width: 1199px) {
+  .header {
+    width: 100%;
+  }
+
+}
+
+.header {
+
+  margin: 0 auto;
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.link__home {
+  margin: 10px 0 0 50px;
+}
+
+.container {
+  display: flex;
+}
+
+.nav__list {
+  list-style: none;
+  text-align: center;
+  line-height: var(--header-height);
+  height: calc(var(--header-height));
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  color: black;
+  font-family: Roboto, sans-serif;
```

Comment:
move to html selector

Reply:
What exactly to transfer to the HTML tag